### DISCUSSION TITLE
lsblk -f  add more output

### DISCUSSION THREAD

**Discussion by mrsteve0924** [util-linux/util-linux#3297] [General]
i like the default output of lsblk -f but sometimes want to see another column.  thought i could do it with lsblk -f -o PARTUUID  but that ends up just showing the PARTUUID.  

any default way to do this? thnaks

**Comment by t-8ch** [comment]
Try `-o +PARTUUID`

**Comment by mrsteve0924** [comment]
that worked.  so simple.  thanks!
